script/Ciel/Day 13/13_01_01_CIEL13_3.txt

```diff
@@ -142,7 +157,9 @@ My head hurts.
 [sha:af8dedb5bd409b707c62327300fa0c5bcc248e0a]{
 -- Page 109, Offset 33475.
 -- <俺|ひと>の頭の中で臆面もなく騒がないでほしい。ただでさえ頭が痛くて倒れそうなのに、こんなに五月蠅いと自分から目を閉じてしまいかねない。
-Can you please stop brazenly kicking up a fuss inside my head? I already feel like I'm going to pass out because of this headache, so if things are this noisy then I might just end up willingly closing my eyes.
+Can you please have some respect and stop kicking up a fuss inside my head?
+ I'm already on the verge of passing out because of this headache,
+ so if you don't pack it in I might just close my eyes and be done with it on purpose.
```

Comment:
The first sentence is a bit too polite, I feel. He's not exactly mincing words. That and the whole "please stop being so noisy, I hate noisy ahhh" doesn't come across very well.
"and be done with it on purpose." sounds a bit too ambiguous for something which is pretty clear in JP.
>I'd like you to stop so unashamedly making a racket in my head.
>...
>so if you're gonna be this noisy, I might just close my eyes and get this over without your help.